Calculator/Program.cs

```diff
@@ -9,14 +9,13 @@ static void Main(string[] args)
         {
             var currentUser = User.GetUsername();
             User.GreetUser(currentUser);
-
-            var calculationChoice = Operations.GetCalculationType(); 
+ 
             var firstNumber = Operations.GetFirstNumber(); 
```

Comment:
You can change the operation class to be an instance instead of a static class.
Why this?
Let's say you want to add A THIRD NUMBER into the mix. You'll need to create another method, add to all the methods you have, unit tests and so on.
How I would improve the Operation class:
I wouldn't make it static
Would create a `readonly List<double> _values = new List<double>()`, this variable will store all values
Remove the first and second number getters and replace with one function `Getvalue`, this function will be responsible to get the value and store it on _values.
Update the other methods to read from this array.

Reply:
👍 Converting to non-static function is a cleaner thing to do. The class will be possible to test, it will be safer as values are not going to be exposed outside of the class.
I'm not sure about `_values` list as code with variables like `firstNumber` is easier to read and usually leads to fewer bugs then `values[0]`.